modules/ROOT/pages/batch-filters-and-batch-aggregator.adoc

```diff
@@ -133,7 +133,7 @@ You can configure the batch aggregator scope to upsert, for example, 100 records
 
 When using a fixed-size aggregator, you can replace, change, or store the payload and variable data of each record.
 
-As stated above, since the batch aggregator is mutable, by adding a foreach scope you can iterate through a fixed-size aggregator block, you can sequentially go over each record's data and persistently store each record's payload and variables. This method of accessing records within the batch aggregator is called sequential access. +
+As stated above, since the Batch Aggregator is mutable, by adding a foreach scope you can iterate through a fixed-size aggregator block, you can sequentially go over each record's data and persistently store each record's payload and variables. This method of accessing records within the Batch Aggregator is called sequential access. +
```

Comment:
I read this sentense multiple times and having trouble to make sense of it. Also the example is not at all helpful.